client/src/component/User.css

```diff
@@ -0,0 +1,32 @@
+.user-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  
+  .user-input {
+    margin-bottom: 10px;
+    padding: 8px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    font-size: 14px;
```

Comment:
this detalis may later change depends on design, for now its good t have them

Reply:
So, I live them for now. Later css style for the whole project will be changed according to our accepted design.